### DISCUSSION TITLE
群辉7.2系统 部署参考

### DISCUSSION THREAD

**Discussion by oddfar** [oddfar/campus-imaotai#92] [分享部署教程]
https://github.com/oddfar/campus-imaotai/discussions/45#discussioncomment-6919752

**Comment by TangTree** [comment]
感谢大佬，按照你的方法，兜兜转转安装运行起来了，就是添加用户的时候，发送验证码一直提示发送验证码错误

**Reply by zhao316678171** [reply]
貌似多点几次发送就可以了，偶尔有这种情况

**Comment by adlofp** [comment]
请问现在出这个问题怎么解决 
![image](https://github.com/oddfar/campus-imaotai/assets/73983769/0144a3cd-198c-4761-9c39-9457329e0c20)

**Reply by peaterRU** [reply]
按照提示，没有到文件就进去目录里面新建一个。

**Comment by thocell** [comment]
⠿ Container campus-mysql   Error                                          2.1s
dependency failed to start: container for service "campus-mysql" is unhealthy
出现这种错误咋办？

**Comment by thocell** [comment]
验证码图片不显示
